Makefile

```diff
@@ -142,3 +140,59 @@ CLIENT_GEN=$(shell go env GOPATH)/bin/client-gen
 else
 CLIENT_GEN=$(shell which client-gen)
 endif
+
+.PHONY: sbom
+sbom: spdx-generator
+	mkdir -p sbom
+	$(SPDX_GENERATOR) -o ./sbom
+
+.PHONY: spdx-generator
+spdx-generator:
+ifeq (, $(shell which spdx-sbom-generator))
+	mkdir -p sbom
+	curl -L https://github.com/spdx/spdx-sbom-generator/releases/download/v0.0.10/spdx-sbom-generator-v0.0.10-linux-amd64.tar.gz -o ./sbom/spdx-sbom-generator-v0.0.10-linux-amd64.tar.gz
+	tar xzvf ./sbom/spdx-sbom-generator-v0.0.10-linux-amd64.tar.gz -C sbom
+SPDX_GENERATOR=./sbom/spdx-sbom-generator
+else
+SPDX_GENERATOR=$(shell which spdx-sbom-generator)
+endif
+
+.PHONY: release
+release:
+	rm -rf release
+	mkdir -p ./release
+
+	# Build the kubectl plugins
+
+	rm -rf ./bin/kubectl-schemahero
+	GOOS=linux GOARCH=amd64 make bin/kubectl-schemahero
+	tar czvf ./release/kubectl-schemahero-linux-amd64.tar.gz ./bin/kubectl-schemahero README.md LICENSE
+
+	rm -rf ./bin/kubectl-schemahero
+	GOOS=linux GOARCH=arm64 make bin/kubectl-schemahero
+	tar czvf ./release/kubectl-schemahero-linux-arm64.tar.gz ./bin/kubectl-schemahero README.md LICENSE
+
+	rm -rf ./bin/kubectl-schemahero
+	GOOS=windows GOARCH=amd64 make bin/kubectl-schemahero
+	tar czvf ./release/kubectl-schemahero-windows-amd64.tar.gz ./bin/kubectl-schemahero README.md LICENSE
+
+	rm -rf ./bin/kubectl-schemahero
+	GOOS=darwin GOARCH=amd64 make bin/kubectl-schemahero
+	tar czvf ./release/kubectl-schemahero-darwin-amd64.tar.gz ./bin/kubectl-schemahero README.md LICENSE
+
+	rm -rf ./bin/kubectl-schemahero
+	GOOS=darwin GOARCH=arm64 make bin/kubectl-schemahero
+	tar czvf ./release/kubectl-schemahero-darwin-arm64.tar.gz ./bin/kubectl-schemahero README.md LICENSE
+
+	# build the docker images for in-cluster
+
+	GOOS=linux GOARCH=amd64 make bin/manager
+	GOOS=linux GOARCH=amd64 make bin/kubectl-schemahero
+	docker build -t schemahero/schemahero:${GITHUB_TAG} -f ./deploy/Dockerfile.schemahero .
+	docker build -t schemahero/manager:${GITHUB_TAG} -f ./deploy/Dockerfile.manager .
+	docker push schemahero/schemahero:${GITHUB_TAG}
+	docker push schemahero/manager:${GITHUB_TAG}
+	cosign attach sbom -sbom ./sbom/bom-go-mod.spdx schemahero/schemahero:${GITHUB_TAG}
+	cosign attach sbom -sbom ./sbom/bom-go-mod.spdx schemahero/manager:${GITHUB_TAG}
+	cosign sign -key ./cosign.key schemahero/schemahero:${GITHUB_TAG}
```

Comment:
Where does this key come from? It's gitignored, and I don't see where in the github actions workflows it gets included

Reply:
Thanks! I'll add it, it will be a GitHub repo secret.

Reply:
@laverya https://github.com/schemahero/schemahero/pull/505/commits/49d274d50af477101ccbcd27b3d798bf2b89536e loads the key from a repo secret into that ephemeral file.